ansible/roles/test/files/ptftests/copp_tests.py

```diff
@@ -106,15 +106,25 @@ def cancel_timeout(self):
         if self.timeout_thr is not None:
             self.timeout_thr.cancel()
             self.timeout_thr = None
-
+    '''
+    Start keeping packets after 1 second to rule out the CBS factor.
+    '''
+    def slipSometime(self):
+         time.sleep(1)
+         self.dataplane.flush()
+    
     def copp_test(self, packet, count, send_intf, recv_intf):
         b_c_0 = self.dataplane.get_counters(*send_intf)
         b_c_1 = self.dataplane.get_counters(*recv_intf)
         b_n_0 = self.dataplane.get_nn_counters(*send_intf)
         b_n_1 = self.dataplane.get_nn_counters(*recv_intf)
-
+        
         start_time=datetime.datetime.now()
 
+        if self.needSlip:
+             t=threading.Thread(target=self.slipSometime)
+             t.start()
+
         for i in xrange(count):
```

Comment:
Let use another approach. We can send the packets for a second first, using send_packet, then count them, discarding the result (probably we can output it), and then run send_packet again. Can you please change?

Reply:
OK.
Follow the suggestions.